elastic/kibana
# [Onboarding] Install integrations and run Agent from auto-detect script

Closes https://github.com/elastic/kibana/issues/183361

### Summary

Integrates [the integrations/install endpoint](https://github.com/elastic/kibana/pull/184167) into the auto-detection flow.

https://github.com/elastic/kibana/assets/793851/19fc9f9d-52fe-48dc-b62e-9abf5ca5b9e0

### How To Test

1. Run Kibana and ES locally
2. In Kibana, generate an API Key which the script will use to install integrations. This will be automated later, as a separate task.
3. In Kibana, go to the quick start Systems Logs flow and copy generated API Key (this will be ingest API key) and onboarding ID from the code snippet.
4. In the terminal run: 
```bash
sudo ./x-pack/plugins/observability_solution/observability_onboarding/public/assets/auto_detect.sh --install-key="{{INSTALL_API_KEY}}"" --ingest-key="{{INGEST_API_KEY}}" --kibana-url="http://localhost:5601/ftw" --id="{{ONBOARDING_ID}}" --ea-version="8.13.4"
```
5. Go through the auto-detect flow (note: the endpoint currently only supports System integration + custom logs)
6. Open Logs explorer and make sure integrations were installed and logs are ingested
